kafka-exporter/src/main/java/io/opentelemetry/contrib/kafka/SpanDataDeserializer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.kafka;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import io.opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest;
+import java.util.Objects;
+import org.apache.kafka.common.errors.SerializationException;
+import org.apache.kafka.common.serialization.Deserializer;
+
+public class SpanDataDeserializer implements Deserializer<ExportTraceServiceRequest> {
+  @Override
+  public ExportTraceServiceRequest deserialize(String topic, byte[] data) {
+    if (Objects.isNull(data)) {
+      return ExportTraceServiceRequest.getDefaultInstance();
```

Comment:
Would you return an empty event? I think if it's not a valid response it should throw. Empty events hide the fact that nothing worthy of processing can be returned

Reply:
In the Deserialiser interface documentation, it states that the 'data' parameter, which represents serialized bytes, may be null. Implementations are advised to handle null by either returning a value or null instead of throwing an exception. That's why the exception is not thrown. I personally opted for returning an empty object to simplify dealing with null values in the callers. If you prefer to receive null values, please let me know.

Reply:
Returning a empty value creates a valid return value that in turn needs to be validated at the client side. This hides the fact that the the incoming event could not be deserialized. 

In other words it transforms the result to a successful event with no content. Default null handling of client side libraries might be a better solution here. I understand that null doesn't seem like the best return value but based what you said about the docs this is the better return type here imo